.gitignore

```diff
@@ -1,23 +1,9 @@
-# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
-
 # dependencies
 /node_modules
-/.pnp
-.pnp.js
-
-# testing
-/coverage
 
 # production
 /build
 
 # misc
 .DS_Store
-.env.local
-.env.development.local
-.env.test.local
-.env.production.local
-
 npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
```

Comment:
Can you add `*~` at the end? It will make the Emacs users (including me) happy.
@dhimmel, do you have any other files on the ignore list?

Reply:
Here's the non-language specific stuff we currently have at https://github.com/manubot/manubot/blob/4505db3fd8a9dc1ba26c36f3fea9accbf02d47d8/.gitignore

```
# System specific files

## Linux
*~
.Trash-*

## macOS
.DS_Store
._*
.Trashes

## Windows
Thumbs.db
[Dd]esktop.ini

## Text Editors
.vscode
```